AlpsSDK/RemoteNotificationManager.swift

```diff
@@ -6,18 +6,24 @@
 //  Copyright © 2017 Alps. All rights reserved.
 //
 
-import Foundation
 import UserNotifications
+import UIKit
+
+protocol RemoteNotificationManagerDelegate: class {
+    func remoteNotificationManager(manager: RemoteNotificationManager, didReceiveNotification: String)
+}
```

Comment:
This protocol could be a general case and could work for websocket too, but I am not sure.